freshservice/assets/logs/freshservice_tests.yaml

```diff
@@ -0,0 +1,157 @@
+id: "freshservice"
```

Comment:
Invalid json here. Multiple instances of booleans and nulls being in error (`None` should be `"None"`, `True` -> `"True"` or `true` etc)